server/bundler/loader-code-splitting.js.template

```diff
@@ -0,0 +1,87 @@
+var config = require( 'config' ),
+	page = require( 'page' ),
+	React = require( 'react' ),
+	activateNextLayoutFocus = require( 'state/ui/layout-focus/actions' ).activateNextLayoutFocus,
+	LoadingError = require( 'layout/error' ),
+	controller = require( 'controller' ),
+	restoreLastSession = require( 'lib/restore-last-path' ).restoreLastSession,
+	preloadHub = require( 'sections-preload' ).hub;
+
+var sections = ${stringSections};
+
+var _loadedSections = {};
+
+function preload( sectionName ) {
+	switch ( sectionName ) {
+		${sectionLoaders}
+	}
```

Comment:
Should we maybe return rejected promise if we had no hit in the switch?

Reply:
Not sure if we can reach this case, how can a section be loaded if we don't create the route using the sections' definition?